test/integration/executor/create_account_test.cpp

```diff
@@ -122,6 +122,30 @@ TEST_P(CreateAccountBasicTest, PrivelegeElevation) {
   checkNoSuchAccount();
 }
 
+/**
+ * Checks that there is no privelege elevation issue via CreateAccount
```

Comment:
```suggestion
 * Checks that there is no privilege elevation issue via CreateAccount
```
That is for the previous test...root allows to create accounts with other permissions.